src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/RerouteTopology.java

```diff
@@ -53,9 +69,12 @@ public StormTopology createTopology() {
 
         TopologyBuilder topologyBuilder = new TopologyBuilder();
 
-        final Integer parallelism = topologyConfig.getParallelism();
+        final Integer parallelism = topologyConfig.getNewParallelism();
```

Comment:
I'm a little bit worried about setting parallelism to 2 in almost all Bolts and spouts. We need to test it carefully

Reply:
Design for this feature also include parallelism support. We are working on testing.